js/mexc.js

```diff
@@ -499,8 +499,10 @@ module.exports = class mexc extends Exchange {
             const quote = this.safeCurrencyCode (quoteId);
             const settle = this.safeCurrencyCode (settleId);
             const state = this.safeString (market, 'state');
+            const fees = this.safeValue (this.fees, 'trading');
```

Comment:
This line does not belong here, if it's not a special case, then it will be handled in setMarkets().